src/components/Card.js

```diff
@@ -1,41 +1,99 @@
+import Api from "../components/Api.js";
+import DeleteCardForm from "./DeleteCardForm.js";
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+    "Content-Type": "application/json",
+  },
+});
```

Comment:
Only **one**  instance of `Api` should be created in the project. No need to do it 2 times